### PR TITLE
Feature/venue info map gp81v2

### PR DESCRIPTION
**Pull request by pbrocks**
This PR addresses Feature 2 of [Venue issue](https://github.com/GatherPress/gatherpress/issues/81)

### CODE REVIEW

**Comment by pbrocks** [comment]
image

forgot to rebuild

File: src/blocks/venue-information/style.scss [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+    justify-content: center;
+}
 .gp-venue {
 	display: flex;
 	flex-direction: column;
 	gap: 0.5rem;
 
 	&__row {
 		display: flex;
+        justify-content: space-between;
```
Comment (mauteri) [inline src/blocks/venue-information/style.scss:19]: fix indent

**Review by mauteri — APPROVED** [review]

File: src/blocks/venue-information/style.scss (hunk 1 above)
Comment (pbrocks) [reply src/blocks/venue-information/style.scss:19]: image

So elfin annoying!!!

**Review by mauteri — CHANGES_REQUESTED** [review]
Following other PR, can this one be closed?

**Comment by pbrocks** [comment]
> Following other PR, can this one be closed?

Absolutely this should be closed.

**Pull request closed without merging**